README.md

```diff
@@ -24,30 +24,32 @@ steps:
     uses: actions/setup-java@v4
```

Comment:
```suggestion
```

I also think, for a usage guide. We don't need this block that setups Java as that is an environment thats set up by the action. 

Here is an example of an example usage of the action.

https://github.com/HassanBahati/firebase-emulator-action-example

[firebase-emulator.yaml](https://github.com/HassanBahati/firebase-emulator-action-example/blob/main/.github/workflows/firebase-emulator.yml)
```sh
name: Firebase Emulator Action Example

on:
  push:
    branches:
      - main

jobs:
  run-emulator:
    runs-on: ubuntu-latest

    steps:
      # Checkout the repository
      - name: Checkout Code
        uses: actions/checkout@v4

      # Start Firebase Emulator Suite
      - name: Start Firebase Emulator Suite
        uses: invertase/firebase-emulator-action@v1.0.0
        with:
          firebase-tools-version: "latest"
          emulators: "auth,firestore,functions,storage,database"
          project-id: "test-project"
          max-retries: "3"
          max-checks: "60"
          wait-time: "1"
          check-port: "8080"
```